utils_test.py

```diff
@@ -0,0 +1,20 @@
+import os
+import unittest
+import utils
+
+class TestUtils(unittest.TestCase):
+    def setUp(self):
+        os.makedirs("data")
```

Comment:
This should have `exist_ok=True` so that it doesn't error in properly set-up dev environments